README.md

```diff
@@ -1,3 +1,103 @@
-# Docker Traceability Plugin
+# CloudBees Docker Traceability Plugin
```

Comment:
Yes, it makes sense if we're planning to add client-side, Docker demos and other such stuff later